prerendering.bs

```diff
@@ -331,6 +331,11 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
   1. Let |request| be a new [=request=] whose [=request/URL=] is |startingURL| and [=request/referrer policy=] is |referrerPolicy|.
 
   1. [=Navigate=] |bc| to |request| with the [=source browsing context=] set to |referrerDoc|'s [=Document/browsing context=].
+
+  1. If |bc| is [=discard|discarded=], [=map/remove=] |referrerDoc|'s [=Document/prerendering browsing contexts map=][(|startingURL|, |referrerPolicy|)].
```

Comment:
Stylistically I think this should be a monkeypatch of "discard", in its own section, similar to https://wicg.github.io/nav-speculation/prerendering.html#no-bad-navs .

The note below is good though and makes sense to have in this algorithm.